content/en/_index.html

```diff
@@ -30,7 +30,9 @@
 {{% blocks/feature image="suitcase" %}}
 #### Run K8s Anywhere
 
-Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud infrastructure, letting you effortlessly move workloads to where it matters to you.
+Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud infrastructure, letting you effortlessly move workloads to where it matters to you. 
+
+To download Kubernetes, visit the [download]({{< relref "/content/en/releases/download.md" >}}) section.
```

Comment:
Why don't we just use `[download](/releases/download/)` ?

Reply:
Oh? Let me try.

Reply:
My bad! sounds good. I'll update this.

Reply:
My bad! sounds good. I'll update this.

Reply:
COuld you please also update the other link on the same page?

Reply:
I'd do the same. Thanks!